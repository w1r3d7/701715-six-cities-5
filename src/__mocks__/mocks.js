export const AuthorizationStatus = {
  AUTH: `AUTH`,
  NO_AUTH: `NO_AUTH`,
};

export const FilterType = {
  POPULAR: `Popular`,
  PRICE_LOW_TO_HIGH: `Price: low to high`,
  PRICE_HIGH_TO_LOW: `Price: high to low`,
  TOP_RATED_FIRST: `Top rated first`,
};

export const offers = [{
  city: {
    name: `Cologne`,
    location: {
      latitude: 50.938361,
      longitude: 6.959974,
      zoom: 13
    }
  },
  title: `The house among olive `,
  rating: 3.8,
  type: `house`,
  bedrooms: 3,
  price: 493,
  description: `The house among olive `,
  location: {
    latitude: 50.957361,
    longitude: 6.9509739999999995,
    zoom: 16
  },
  id: 1,
  bedroomsCount: 3,
  facilities: [
    `Laptop friendly workspace`,
    `Washer`,
    `Towels`,
    `Fridge`,
    `Air conditioning`,
    `Baby seat`,
    `Breakfast`
  ],
  photosUrl: [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`
  ],
  isPremium: false,
  isInBookmark: false,
  hostName: `Angelina`,
  hostDescription: `Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.`,
  hostAvatar: `img/avatar-angelina.jpg`,
  isHostPremium: true,
  maxCapacity: 4,
  previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`
},
{
  city: {
    name: `Brussels`,
    location: {
      latitude: 50.846557,
      longitude: 4.351697,
      zoom: 13
    }
  },
  title: `Amazing and Extremely Central Flat`,
  rating: 4.3,
  type: `apartment`,
  bedrooms: 3,
  price: 323,
  description: `Amazing and Extremely Central Flat`,
  location: {
    latitude: 50.839557,
    longitude: 4.346697,
    zoom: 16
  },
  id: 2,
  bedroomsCount: 3,
  facilities: [
    `Towels`,
    `Baby seat`,
    `Laptop friendly workspace`,
    `Dishwasher`,
    `Washing machine`,
    `Air conditioning`,
    `Washer`,
    `Coffee machine`,
    `Breakfast`,
    `Fridge`
  ],
  photosUrl: [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/20.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/18.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`
  ],
  isPremium: false,
  isInBookmark: false,
  hostName: `Angelina`,
  hostDescription: `This is a place for dreamers to reset, reflect, and create. Designed with a 'slow' pace in mind, our hope is that you enjoy every part of your stay; from making local coffee by drip in the morning, choosing the perfect record to put on as the sun sets.`,
  hostAvatar: `img/avatar-angelina.jpg`,
  isHostPremium: true,
  maxCapacity: 8,
  previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`
},
{
  city: {
    name: `Brussels`,
    location: {
      latitude: 50.846557,
      longitude: 4.351697,
      zoom: 13
    }
  },
  title: `The house among olive `,
  rating: 4.6,
  type: `apartment`,
  bedrooms: 4,
  price: 218,
  description: `The house among olive `,
  location: {
    latitude: 50.865556999999995,
    longitude: 4.371696999999999,
    zoom: 16
  },
  id: 3,
  bedroomsCount: 4,
  facilities: [
    `Washer`,
    `Laptop friendly workspace`,
    `Breakfast`,
    `Air conditioning`,
    `Baby seat`
  ],
  photosUrl: [
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/12.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/6.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/8.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/7.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg`,
    `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg`
  ],
  isPremium: false,
  isInBookmark: false,
  hostName: `Angelina`,
  hostDescription: `Relax, rejuvenate and unplug in this ultimate rustic getaway experience in the country. In our beautiful screened Pondhouse, you can gaze at the stars and listen to the sounds of nature from your cozy warm bed.`,
  hostAvatar: `img/avatar-angelina.jpg`,
  isHostPremium: true,
  maxCapacity: 7,
  previewImage: `https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg`
}
];


export const reviews = [{
  "id": 1,
  "rating": 4,
  "date": `2020-11-15T13:58:19.494Z`,
  "reviewText": `421421`,
  "photoUrl": `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/3.jpg`,
  "name": `xyz`,
  "isUserPremium": false,
  "userId": 1
}, {
  "id": 2,
  "rating": 3,
  "date": `2020-10-15T13:58:19.494Z`,
  "reviewText": `1111`,
  "photoUrl": `https://assets.htmlacademy.ru/intensives/javascript-3/avatar/4.jpg`,
  "name": `xyza`,
  "isUserPremium": true,
  "userId": 2
}
];
