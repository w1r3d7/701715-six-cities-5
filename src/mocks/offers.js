const getRating = () => {
  return String((Math.random() * 5).toFixed(2));
};

const getRandomItemsCount = (array) => {
  const shuffled = array.sort(() => 0.5 - Math.random());
  return shuffled.slice(0, Math.floor(Math.random() * array.length));
};

const reviews = [
  {
    name: `Oleg`,
    id: 11,
    photoUrl: `/img/avatar-max.jpg`,
    reviewText: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt laudantium tempore veritatis? A
      asperiores debitis deserunt maxime molestiae nihil obcaecati, odio officiis quos repellendus sed, sit ut veniam?
      Eligendi, illo. `,
    rating: getRating(),
    date: new Date().toDateString(),
  },
  {
    name: `Petr`,
    id: 21,
    photoUrl: `/img/avatar-max.jpg`,
    reviewText: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt laudantium tempore veritatis? A
      asperiores debitis deserunt maxime molestiae nihil obcaecati, odio officiis quos repellendus sed, sit ut veniam?
      Eligendi, illo. `,
    rating: getRating(),
    date: new Date().toDateString(),
  },
  {
    name: `Max`,
    id: 31,
    photoUrl: `/img/avatar-max.jpg`,
    reviewText: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt laudantium tempore veritatis? A
      asperiores debitis deserunt maxime molestiae nihil obcaecati, odio officiis quos repellendus sed, sit ut veniam?
      Eligendi, illo. `,
    rating: getRating(),
    date: new Date().toDateString(),
  },
  {
    name: `Angelina`,
    id: 41,
    photoUrl: `/img/avatar-angelina.jpg`,
    reviewText: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Incidunt laudantium tempore veritatis? A
      asperiores debitis deserunt maxime molestiae nihil obcaecati, odio officiis quos repellendus sed, sit ut veniam?
      Eligendi, illo. `,
    rating: getRating(),
    date: new Date().toDateString(),
  },
];

export default [
  {
    id: 1,
    city: `Amsterdam`,
    photosUrl: [`/img/apartment-01.jpg`, `/img/apartment-small-03.jpg`, `/img/apartment-small-04.jpg`],
    type: `Apartment`,
    price: 120,
    description: `Beautiful & luxurious studio at great location`,
    isPremium: true,
    isInBookmark: false,
    facilities: [
      `Wi-Fi`,
      `Towels`,
      `Washing machine`,
      `Heating`,
    ],
    rating: getRating(),
    bedroomsCount: 3,
    maxCapacity: 4,
    hostName: `Angelina`,
    hostAvatar: `/img/avatar-angelina.jpg`,
    isHostPremium: true,
    hostDescription: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolor eius et facilis fuga mollitia
      praesentium quasi qui quibusdam quo recusandae repellat, repellendus rerum saepe sapiente sequi unde velit vero
      voluptatem.`,
    reviews: getRandomItemsCount(reviews),
  },
  {
    id: 2,
    city: `Amsterdam`,
    photosUrl: [`/img/apartment-02.jpg`, `/img/apartment-small-03.jpg`, `/img/apartment-small-04.jpg`],
    type: `House`,
    price: 40,
    description: `Great location`,
    isPremium: true,
    isInBookmark: true,
    facilities: [
      `Wi-Fi`,
      `Towels`,
      `Washing machine`,
      `Heating`,
    ],
    rating: getRating(),
    bedroomsCount: 1,
    maxCapacity: 2,
    hostName: `Igor`,
    hostAvatar: `/img/avatar-max.jpg`,
    isHostPremium: false,
    hostDescription: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Blanditiis doloremque in iure
      laborum magnam placeat quasi quidem quo! Assumenda error fuga iste laborum molestiae nam odit quia repellendus sed
      soluta.`,
    reviews: getRandomItemsCount(reviews),
  },
  {
    id: 3,
    city: `Amsterdam`,
    photosUrl: [`/img/apartment-03.jpg`, `/img/apartment-small-03.jpg`, `/img/apartment-small-04.jpg`],
    type: `Villa`,
    price: 4000,
    description: `Nice, cozy, warm big bed apartment`,
    isPremium: false,
    isInBookmark: false,
    facilities: [
      `Washing machine`,
      `Towels`,
      `Heating`,
      `Coffee machine`,
      `Baby seat`,
    ],
    rating: getRating(),
    bedroomsCount: 4,
    maxCapacity: 5,
    hostName: `Oleg`,
    hostAvatar: `/img/avatar-max.jpg`,
    isHostPremium: true,
    hostDescription: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Blanditiis doloremque in iure
      laborum magnam placeat quasi quidem quo! Assumenda error fuga iste laborum molestiae nam odit quia repellendus sed
      soluta.`,
    reviews: getRandomItemsCount(reviews),
  },
  {
    id: 4,
    city: `Amsterdam`,
    photosUrl: [`/img/room.jpg`, `/img/apartment-small-03.jpg`, `/img/apartment-small-04.jpg`],
    type: `Private Room`,
    price: 4,
    description: `Wood and stone place`,
    isPremium: false,
    isInBookmark: true,
    facilities: [
      `Kitchen`,
      `Dishwasher`,
      `Cabel TV`,
      `Fridge`,
    ],
    rating: getRating(),
    bedroomsCount: 1,
    maxCapacity: 1,
    hostName: `Oleg`,
    hostAvatar: `/img/avatar-max.jpg`,
    isHostPremium: false,
    hostDescription: `Lorem ipsum dolor sit amet, consectetur adipisicing elit. Blanditiis doloremque in iure
      laborum magnam placeat quasi quidem quo! Assumenda error fuga iste laborum molestiae nam odit quia repellendus sed
      soluta.`,
    reviews: getRandomItemsCount(reviews),
  },
];
