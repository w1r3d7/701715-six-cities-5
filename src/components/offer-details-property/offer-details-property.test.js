import React from 'react';
import renderer from 'react-test-renderer';

import OfferDetailsProperty from './offer-details-property';

import {offers} from '../../__mocks__/mocks';

const [offer] = offers;

test(`OfferDetailsProperty render correctly`, () => {
  const tree = renderer
    .create(
        <OfferDetailsProperty offer={offer} nearbyOffers={offers} />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
