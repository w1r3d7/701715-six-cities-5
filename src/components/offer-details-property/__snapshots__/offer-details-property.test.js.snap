// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferDetailsProperty render correctly 1`] = `
<section
  className="property"
>
  <div
    className="property__gallery-container container"
  >
    <div
      className="property__gallery"
    >
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/15.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/16.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/9.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/10.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/4.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/19.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/14.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/3.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/2.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/11.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/1.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/17.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/5.jpg"
        />
      </div>
      <div
        className="property__image-wrapper"
      >
        <img
          alt="house"
          className="property__image"
          src="https://assets.htmlacademy.ru/intensives/javascript-3/hotel/13.jpg"
        />
      </div>
    </div>
  </div>
  <div
    className="property__container container"
  >
    <div
      className="property__wrapper"
    >
      <div
        className="property__name-wrapper"
      >
        <h1
          className="property__name"
        >
          The house among olive 
        </h1>
        <button
          className="property__bookmark-button button "
          onClick={[Function]}
          type="button"
        >
          <svg
            className="property__bookmark-icon"
            height="33"
            width="31"
          >
            <use
              xlinkHref="#icon-bookmark"
            />
          </svg>
          <span
            className="visually-hidden"
          >
            To bookmarks
          </span>
        </button>
      </div>
      <div
        className="property__rating rating"
      >
        <div
          className="property__stars rating__stars"
        >
          <span
            style={
              Object {
                "width": "76%",
              }
            }
          />
          <span
            className="visually-hidden"
          >
            Rating
          </span>
        </div>
        <span
          className="property__rating-value rating__value"
        >
          3.8
        </span>
      </div>
      <ul
        className="property__features"
      >
        <li
          className="property__feature property__feature--entire"
        >
          house
        </li>
        <li
          className="property__feature property__feature--bedrooms"
        >
          3 bedrooms
        </li>
        <li
          className="property__feature property__feature--adults"
        >
          Max 
          4
           adults
        </li>
      </ul>
      <div
        className="property__price"
      >
        <b
          className="property__price-value"
        >
          €
          493
        </b>
        <span
          className="property__price-text"
        >
           night
        </span>
      </div>
      <div
        className="property__inside"
      >
        <h2
          className="property__inside-title"
        >
          What's inside
        </h2>
        <ul
          className="property__inside-list"
        >
          <li
            className="property__inside-item"
          >
            Laptop friendly workspace
          </li>
          <li
            className="property__inside-item"
          >
            Washer
          </li>
          <li
            className="property__inside-item"
          >
            Towels
          </li>
          <li
            className="property__inside-item"
          >
            Fridge
          </li>
          <li
            className="property__inside-item"
          >
            Air conditioning
          </li>
          <li
            className="property__inside-item"
          >
            Baby seat
          </li>
          <li
            className="property__inside-item"
          >
            Breakfast
          </li>
        </ul>
      </div>
      <div
        className="property__host"
      >
        <h2
          className="property__host-title"
        >
          Meet the host
        </h2>
        <div
          className="property__host-user user"
        >
          <div
            className="property__avatar-wrapper user__avatar-wrapper property__avatar-wrapper--pro"
          >
            <img
              alt="Host avatar"
              className="property__avatar user__avatar"
              height="74"
              src="img/avatar-angelina.jpg"
              width="74"
            />
          </div>
          <span
            className="property__user-name"
          >
            Angelina
          </span>
        </div>
        <div
          className="property__description"
        >
          <p
            className="property__text"
          >
            Discover daily local life in city center, friendly neighborhood, clandestine casino, karaoke, old-style artisans, art gallery and artist studio downstairs.
          </p>
        </div>
      </div>
      <section
        className="property__reviews reviews"
      >
        <h2
          className="reviews__title"
        >
          Reviews ·
          <span
            className="reviews__amount"
          >
            2
          </span>
        </h2>
        <ul
          className="reviews__list"
        >
          <li
            className="reviews__item"
          >
            <div
              className="reviews__user user"
            >
              <div
                className="reviews__avatar-wrapper user__avatar-wrapper"
              >
                <img
                  alt="Avatar xyz"
                  className="reviews__avatar user__avatar"
                  height="54"
                  src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/3.jpg"
                  width="54"
                />
              </div>
              <span
                className="reviews__user-name"
              >
                xyz
              </span>
            </div>
            <div
              className="reviews__info"
            >
              <div
                className="reviews__rating rating"
              >
                <div
                  className="reviews__stars rating__stars"
                >
                  <span
                    style={
                      Object {
                        "width": "80%",
                      }
                    }
                  />
                  <span
                    className="visually-hidden"
                  >
                    Rating
                  </span>
                </div>
              </div>
              <p
                className="reviews__text"
              >
                421421
              </p>
              <time
                className="reviews__time"
                dateTime="2020-11-15T13:58:19.494Z"
              >
                November 2020
              </time>
            </div>
          </li>
          <li
            className="reviews__item"
          >
            <div
              className="reviews__user user"
            >
              <div
                className="reviews__avatar-wrapper user__avatar-wrapper"
              >
                <img
                  alt="Avatar xyza"
                  className="reviews__avatar user__avatar"
                  height="54"
                  src="https://assets.htmlacademy.ru/intensives/javascript-3/avatar/4.jpg"
                  width="54"
                />
              </div>
              <span
                className="reviews__user-name"
              >
                xyza
              </span>
            </div>
            <div
              className="reviews__info"
            >
              <div
                className="reviews__rating rating"
              >
                <div
                  className="reviews__stars rating__stars"
                >
                  <span
                    style={
                      Object {
                        "width": "60%",
                      }
                    }
                  />
                  <span
                    className="visually-hidden"
                  >
                    Rating
                  </span>
                </div>
              </div>
              <p
                className="reviews__text"
              >
                1111
              </p>
              <time
                className="reviews__time"
                dateTime="2020-10-15T13:58:19.494Z"
              >
                October 2020
              </time>
            </div>
          </li>
        </ul>
        <form
          action="#"
          className="reviews__form form"
          method="post"
          onSubmit={[Function]}
        >
          <label
            className="reviews__label form__label"
            htmlFor="review"
          >
            Your review
          </label>
          <div
            className="reviews__rating-form form__rating"
          >
            <input
              className="form__rating-input visually-hidden"
              disabled="disabled"
              id="5-stars"
              name="rating"
              required={true}
              type="radio"
              value="5"
            />
            <label
              className="reviews__rating-label form__rating-label"
              htmlFor="5-stars"
              title="perfect"
            >
              <svg
                className="form__star-image"
                height="33"
                width="37"
              >
                <use
                  xlinkHref="#icon-star"
                />
              </svg>
            </label>
            <input
              className="form__rating-input visually-hidden"
              disabled="disabled"
              id="4-stars"
              name="rating"
              required={true}
              type="radio"
              value="4"
            />
            <label
              className="reviews__rating-label form__rating-label"
              htmlFor="4-stars"
              title="good"
            >
              <svg
                className="form__star-image"
                height="33"
                width="37"
              >
                <use
                  xlinkHref="#icon-star"
                />
              </svg>
            </label>
            <input
              className="form__rating-input visually-hidden"
              disabled="disabled"
              id="3-stars"
              name="rating"
              required={true}
              type="radio"
              value="3"
            />
            <label
              className="reviews__rating-label form__rating-label"
              htmlFor="3-stars"
              title="not bad"
            >
              <svg
                className="form__star-image"
                height="33"
                width="37"
              >
                <use
                  xlinkHref="#icon-star"
                />
              </svg>
            </label>
            <input
              className="form__rating-input visually-hidden"
              disabled="disabled"
              id="2-stars"
              name="rating"
              required={true}
              type="radio"
              value="2"
            />
            <label
              className="reviews__rating-label form__rating-label"
              htmlFor="2-stars"
              title="badly"
            >
              <svg
                className="form__star-image"
                height="33"
                width="37"
              >
                <use
                  xlinkHref="#icon-star"
                />
              </svg>
            </label>
            <input
              className="form__rating-input visually-hidden"
              disabled="disabled"
              id="1-stars"
              name="rating"
              required={true}
              type="radio"
              value="1"
            />
            <label
              className="reviews__rating-label form__rating-label"
              htmlFor="1-stars"
              title="terribly"
            >
              <svg
                className="form__star-image"
                height="33"
                width="37"
              >
                <use
                  xlinkHref="#icon-star"
                />
              </svg>
            </label>
          </div>
          <textarea
            className="reviews__textarea form__textarea"
            disabled="disabled"
            id="review"
            maxLength="300"
            minLength="50"
            name="review"
            placeholder="Tell how was your stay, what you like and what can be improved"
          />
          <div
            className="reviews__button-wrapper"
          >
            <p
              className="reviews__help"
            >
              To submit review please make sure to set 
              <span
                className="reviews__star"
              >
                rating
              </span>
               and describe your stay with at least 
              <b
                className="reviews__text-amount"
              >
                50 characters
              </b>
              .
            </p>
            <button
              className="reviews__submit form__submit button"
              disabled="disabled"
              type="submit"
            >
              Submit
            </button>
          </div>
        </form>
      </section>
    </div>
  </div>
  <section
    className="property__map map"
    id="map"
  />
</section>
`;
